hawk/src/main/java/com/orhanobut/hawk/Hawk.java

```diff
@@ -18,6 +18,10 @@ private Hawk() {
     // no instance
   }
 
+  public enum Status {
+    NOT_INITIALISED, INITIALISED
```

Comment:
do we need status? I think a boolean would be enough. I can't think any other status really.